### DISCUSSION TITLE
addons don't deploy

### DISCUSSION THREAD

**Discussion by fredbouin** [statamic/cms#4568] [Troubleshooting] [answered]
Any new addon I install in dev doesn't deploy to the production environment. If I look at the deployment log it looks like the files are being pulled down from the repo, here's a log from forge:

```
Fast-forward
 composer.json                                |   1 +
 composer.lock                                |  47 ++++++++++++++++++++++++++-
 public/vendor/radnight/css/radnight.css      |   1 +
 public/vendor/radnight/img/trans-bg-dark.gif | Bin 0 -> 58 bytes
 resources/views/layout.antlers.html          |   4 +--
 5 files changed, 50 insertions(+), 3 deletions(-)
 create mode 100644 public/vendor/radnight/css/radnight.css
 create mode 100644 public/vendor/radnight/img/trans-bg-dark.gif
```

I'm not seeing the addon istalled in cp though, and it looks like I'm getting the same problem updating previously installed and deployed addons, the update is not showing up in the production end.

**Comment by jasonvarga** [comment, marked as answer]
Are you running `composer install` when you deploy?

**Reply by fredbouin** [reply]
problem solved, thanks for that
